src/diffusers/pipelines/musicldm/pipeline_musicldm.py

```diff
@@ -51,7 +51,7 @@
         >>> import torch
         >>> import scipy
 
-        >>> repo_id = "cvssp/audioldm-s-full-v2"
+        >>> repo_id = "sanchit-gandhi/musicldm-full"
```

Comment:
Hmm why does the full-v2 not work? We should try to always use the official checkpoints if possible